### PR TITLE
Add `aftewards` misspelling

### PR DESCRIPTION
**Pull request by korverdev**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2328,6 +2328,7 @@ aformentioned->aforementioned
 afrer->after
 afterall->after all
 afterw->after
+aftewards->afterwards
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2331]: For consistency with other misspelling fixes, I would suggest:
```suggestion
afteward->afterward
aftewards->afterwards
 ```

Comment (korverdev) [reply codespell_lib/data/dictionary.txt:2331]: Appreciate it, @DimitriPapadopoulos!

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:2331]: Could you add `afteward` to your pull request ? I cannot add it using the GitHub web interface, it cannot handle large files and prints nonsensical error messages.

Comment (korverdev) [reply codespell_lib/data/dictionary.txt:2331]: Sorry about that. I've been away from my computer. I have added that commit. This should be ready for you to re-review.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @korverdev.

**Pull request merged**